wix/main.wxs

```diff
@@ -0,0 +1,240 @@
+              the license sidecar file from the installer.
+            -->
+            <ComponentRef Id='License'/>
+			<ComponentRef Id='batch0'/>            
+            <ComponentRef Id='binary0'/>
+			<ComponentRef Id="ApplicationShortcut"/>
+			<ComponentRef Id="Readme"/>
+
+            <Feature
+                Id='Environment'
+                Title='PATH Environment Variable'
+                Description='Add the install location of the [ProductName] executable to the PATH system environment variable. This allows the [ProductName] executable to be called from any location.'
+                Level='1'
+                Absent='allow'>
+                <ComponentRef Id='Path'/>
+            </Feature>
+        </Feature>
+
+        <SetProperty Id='ARPINSTALLLOCATION' Value='[APPLICATIONFOLDER]' After='CostFinalize'/>
+
+        <!--
+          Disabling the custom product icon for the application in the
+          Add/Remove Programs control panel requires commenting out or
+          removing the following `Icon` and `Property` tags.
+        -->
+        <Icon Id='ProductICO' SourceFile='QLDBShell.ico'/>
+        <Property Id='ARPPRODUCTICON' Value='ProductICO' />
+		
+        
+        <!--
+          Adding a URL to Add/Remove Programs control panel listing for the
+          application is a two step process:
+
+          1. Uncomment the following `Property` tag with the "ARPHELPLINK" Id
+             attribute value.
+          2. Change the value for `Value` attribute of the following
+             `Property` tag to a valid URL.
+        -->
+        <!--<Property Id='ARPHELPLINK' Value='ChangeMe'/>-->
+
+        <UI>
+				
+			<!--
+			<UIRef Id="WixUI_Mondo" />
+			<UIRef Id="WixUI_ErrorProgressText" />
+			-->			
+			
+            <UIRef Id='WixUI_FeatureTree'/>
+			<UIRef Id="WixUI_ErrorProgressText" />
+			          
+            <!--
+              Enabling the EULA dialog in the installer is a three step process:
+
+                1. Comment out or remove the two `Publish` tags that follow the
+                   `WixVariable` tag.
+                2. Uncomment the `<WixVariable Id='WixUILicenseRtf' Value='Path\to\Eula.rft'>` tag futher down
+                3. Replace the `Value` attribute of the `WixVariable` tag with
+                   the path to a RTF file that will be used as the EULA and
+                   displayed in the license agreement dialog.
+            -->
+            <!--
+			<Publish Dialog='WelcomeDlg' Control='Next' Event='NewDialog' Value='CustomizeDlg' Order='99'>1</Publish>
+			
+			<Publish Dialog='CustomizeDlg' Control='Back' Event='NewDialog' Value='WelcomeDlg' Order='99'>1</Publish>
+			-->
+
+        </UI>
+
+        
+        <!--
+          Enabling the EULA dialog in the installer requires uncommenting
+          the following `WixUILicenseRTF` tag and changing the `Value`
+          attribute.
+        -->
+        <WixVariable Id='WixUILicenseRtf' Value='AWS_Sample_EULA.rtf'/>
```

Comment:
I believe this should be removed per our standup discussion this morning.

Reply:
It's now removed.  It was there for testing last week.